src/microbe_stage/OrganelleDefinition.cs

```diff
@@ -496,6 +516,31 @@ public void Resolve(SimulationParameters parameters)
         }
     }
 
+    /// <summary>
+    ///   A bbcode string containing all the unlock conditions for this organelle.
+    /// </summary>
+    public LocalizedStringBuilder GenerateUnlockRequirementsText(GameProperties game)
+    {
+        LocalizedStringBuilder unlockRequirements = new();
```

Comment:
Would it be possible for this to take in a string builder as a parameter to allow reusing string builder objects?